### PR TITLE
 fix: view create course 

### PR DESCRIPTION
**Pull request by saidabdul80**
sorted to admin side nav and content was make to be vue component and as well backend connected

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /node_modules
 /public/hot
 /public/storage
+/app/Http/Controllers/Api
```
Comment (Auwalms) [inline .gitignore]: Won't this affect @bynalab works? Not sure it's already detached @saidabdul80

File: resources/js/components/createCourse.vue [hunk 2]
```diff
@@ -535,17 +535,40 @@
...
+								}).then((result) => {								  
+								  if (result.isConfirmed) {
+								    location.reload();
+								  } else if (result.isDenied) {								    
+								  }
+								})
+				   			}
+			            	//console.log(response);			   				           
 			            }, function(e) {		  
+			            	console.log(e.status)
```
Comment (Auwalms) [inline resources/js/components/createCourse.vue:571]: You want send console log to Production? Fear God 🤣🤣

File: resources/js/plugin/plugin.js [hunk 3]
```diff
@@ -37,7 +39,82 @@ export default {
       		$('#toastdesc').html(msg);
       		x.addClass("show");
       		setTimeout(function(){x.removeClass("show"); }, 5000);
+  		},
+  		frontendLogout(){
```
Comment (Auwalms) [inline resources/js/plugin/plugin.js:43]: You sha like long variable name. Why not just logout()

**Review by Auwalms — APPROVED** [review]
Looks okay, check confirm the gitignore doesn't affect Abdulsalam.

**Review by abdullahikawu — APPROVED** [review]
Kindly respond to @Auwalms's comment, then I approve

**Pull request merged**